{"name": "Возврат ошибки при неправильном значение поля", "status": "passed", "description": "Проверка, что при неправильном значение поля, возвращается кода 404 и текст ошибки в теле ответа", "steps": [{"name": "Отправка POST-запроса для авторизации курьера", "status": "passed", "start": 1739105991879, "stop": 1739105992008}, {"name": "Проверка, что код ответа на создание заказа 404", "status": "passed", "start": 1739105992008, "stop": 1739105992008}, {"name": "Проверка, что в теле ответа содержится сообщение об ошибки", "status": "passed", "start": 1739105992009, "stop": 1739105992009}], "attachments": [{"name": "log", "source": "20f0583a-290d-4a15-8cc3-62d1ee184033-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "f5becc80-8d76-4511-ba06-bd9fefb2a4c4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_field", "value": "'login'"}], "start": 1739105991879, "stop": 1739105992009, "uuid": "60bff401-6ec1-4a05-b3d3-6e7f69b246e0", "historyId": "a85fa411bbe7d7019aeaa77e04a31c2c", "testCaseId": "51569ab39f17883721bc826a2a78674c", "fullName": "tests.test_auth_courier.TestAuthCourier#test_courier_login_incorrect_field_returns_error", "labels": [{"name": "story", "value": "Логин курьера в системе"}, {"name": "feature", "value": "Работа с курьером"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_courier"}, {"name": "subSuite", "value": "TestAuthCourier"}, {"name": "host", "value": "bnovo-nb-0056"}, {"name": "thread", "value": "24152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_courier"}]}